examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
@bgyarfas Just checking timelines, is this something you can revert in the next few hours? If not, absolutely no worries, but then I  think we will have to push it to `0.12.3` in that case, this is the last PR we are looking at for `0.12.2`, and we need to start the release process today.